script.js

```diff
@@ -85,15 +117,49 @@ function createEpisodeListItem(episode) {
   return episodeListItem;
 }
 
-function fillEpisodeList() {
+function populateShowSelector() {
+  const shows = state.shows;
+  for (const show of shows) {
+    const showListItem = createShowListItem(show);
+    showSelector.append(showListItem);
+    if (show.id == state.selectedShowId) {
+      showSelector.value = show.id;
+    }
+  }
+}
+
+function populateEpisodeSelector() {
+    episodeSelector.textContent = "";
+    const defaultOption = document.createElement("option");
+    defaultOption.selected = true;
+    defaultOption.disabled = true;
+    defaultOption.innerHTML = "Select an option";
+    document.getElementById("episode-selector").append(defaultOption);
   const episodes = state.episodes;
   for (e of episodes) {
```

Comment:
Can you rename `e` variable to more descriptive name?

Reply:
Oops, looks like I updated them in my Repo but forgot to do that here, but I have made the changes now